includes/polling/os/fabos.inc.php

```diff
@@ -1,4 +1,199 @@
 <?php
 
 $version  = trim(snmp_get($device, '1.3.6.1.4.1.1588.2.1.1.1.1.6.0', '-Ovq'), '"');
-$hardware = trim(snmp_get($device, 'ENTITY-MIB::entPhysicalDescr.1', '-Ovq'), '"');
+$gethardware = trim(snmp_get($device, 'SNMPv2-SMI::mib-2.75.1.1.4.1.3.1', '-Ovq'), '"');
+$revboard = str_replace("SNMPv2-SMI::enterprises.1588.2.1.1.", "", $gethardware);
+if(strpos($revboard, ".") !== false) {
+	$getid = strstr(str_replace($revboard, "", $gethardware), ".", true);
+} else {
+	$getid = $revboard;
+}
+
+switch($getid)
```

Comment:
You should move this logic to: includes/rewrites.php as per other vendors hardware rewriting.